quickstart/101-front-door-standard-premium/readme.md

```diff
@@ -0,0 +1,298 @@
+# Azure Front Door Standard/Premium
+
+This template deploys an [Azure Front Door Standard/Premium profile](https://registry.terraform.io/providers/hashicorp/azurerm/latest/docs/resources/cdn_frontdoor_profile) with an App Service origin.
+
+## Resources
+
+| Terraform Resource Type | Description |
+| - | - |
+| `azurerm_resource_group` | The resource group for all the deployed resources.|
+| `azurerm_cdn_frontdoor_profile` | The Front Door profile. |
+| `azurerm_cdn_frontdoor_endpoint` | The Front Door endpoint. |
+| `azurerm_cdn_frontdoor_origin_group` | The Front Door origin group. |
+| `azurerm_cdn_frontdoor_origin` | The Front Door origin, which refers to the App Service app. |
+| `azurerm_cdn_frontdoor_route` | The Front Door route. |
+| `azurerm_service_plan` | The App Service plan. |
+| `azurerm_windows_web_app` | The App Service app. |
+
+## Variables
+
```

Comment:
Would be good to document the defaults here as well.

Reply:
Updated. Was this what you had in mind?